src/java/htsjdk/samtools/liftover/LiftOver.java

```diff
@@ -139,6 +143,29 @@ public Interval liftOver(final Interval interval, final double liftOverMinMatch)
         return ret;
     }
 
+    /**
+     * @return the set of destination contigs for each source contig in the chains file.
+     */
+    public Map<String, Set<String>> getContigMap() {
```

Comment:
since the chainfile is a member, wouldn't it make sense to store the contigMap as a member as well (initialize to null) ? otherwise you will recalculate this map every time someone calls getContigMap.